data/filedefs/filetypes.asciidoc

```diff
@@ -1,35 +1,36 @@
 # For complete documentation of this file, please see Geany's main documentation
 [styling]
-# no syntax highlighting yet
+# Edit these in the colorscheme .conf file instead
+default=default
+strong=default,bold
+emphasis=default,italic
+header1=keyword_1
+header2=keyword_1
+header3=keyword_1
+header4=keyword_1
+header5=keyword_1
+header6=keyword_1
+ulist_item=tag_unknown
+olist_item=tag_unknown
+blockquote=tag_unknown
+strikeout=tag_unknown
```

Comment:
It's important that filetypes.asciidoc assigns mappings according to whatever you enter in the header file `highlightingmappings.h`. Pasted for reference below

So it seems that you copied too much from filetypes.markdown, `strikeout` is defined there but not for asciidoc. I think you need to cleanup again.

```

	{ SCE_ASCIIDOC_DEFAULT,		"default",		FALSE },
	{ SCE_ASCIIDOC_STRONG1,		"strong",		FALSE },
	{ SCE_ASCIIDOC_STRONG2,		"strong",		FALSE },
	{ SCE_ASCIIDOC_EM1,			"emphasis",		FALSE },
	{ SCE_ASCIIDOC_EM2,			"emphasis",		FALSE },
	{ SCE_ASCIIDOC_HEADER1,		"header1",		FALSE },
	{ SCE_ASCIIDOC_HEADER2,		"header2",		FALSE },
	{ SCE_ASCIIDOC_HEADER3,		"header3",		FALSE },
	{ SCE_ASCIIDOC_HEADER4,		"header4",		FALSE },
	{ SCE_ASCIIDOC_HEADER5,		"header5",		FALSE },
	{ SCE_ASCIIDOC_HEADER6,		"header6",		FALSE },
	{ SCE_ASCIIDOC_ULIST_ITEM,	"ulist_item",	FALSE },
	{ SCE_ASCIIDOC_OLIST_ITEM,	"olist_item",	FALSE },
	{ SCE_ASCIIDOC_BLOCKQUOTE,	"blockquote",	FALSE },
	{ SCE_ASCIIDOC_LINK,		"link",			FALSE },
	{ SCE_ASCIIDOC_CODEBK,		"code",			FALSE },
	{ SCE_ASCIIDOC_PASSBK,		"passthrough",	FALSE },
	{ SCE_ASCIIDOC_COMMENT,		"comment",		FALSE },
	{ SCE_ASCIIDOC_COMMENTBK,	"comment",		FALSE },
	{ SCE_ASCIIDOC_LITERAL,		"literal",		FALSE },
	{ SCE_ASCIIDOC_LITERALBK,	"literal",		FALSE },
	{ SCE_ASCIIDOC_ATTRIB,		"attrib",		FALSE },
	{ SCE_ASCIIDOC_ATTRIBVAL,	"attribval",	FALSE },
	{ SCE_ASCIIDOC_MACRO,		"macro",		FALSE }
```